extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -24,6 +35,10 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 	function createProvider(logGroup, providerName, src, slotMap, extra) {
 		extra = extra || {};
 
+		if (extra.overrideSizesPerCountry && extra.overrideSizesPerCountry[country]) {
+			overrideSizes(slotMap, extra.overrideSizesPerCountry[country]);
```

Comment:
I think the initial idea was to make some params required and optional (`extra`). I agree that it's worth to rethink this design, but not sure if in scope of this ticket - it's a bit sensitive area and it's Q4 :)